### ISSUE TITLE
Estimates User Issues

### ISSUE THREAD

**Issue by alizard2** [microsoft/azure-boards-estimate#163]
1. Anyone can click "reveal"
2. Anyone can change the view for everyone else by clicking into a different user story.
3. If anyone moves to a different user story before the vote is saved, all votes are wiped.   

I've used this tool for a while now and just introduced it to our team at my current job.  It can get frustrating when someone clicks away from the current story causing already provided votes on the current story to get wiped.  Votes already made need to get saved on the story at all times.

**Comment by AminTi** [comment]
Thanks for reporting

**Comment by Blackbaud-MaryRudisill** [comment]
Well, this is a breaking change. It would have been better if alizard2 had just trained the team members to not click on other user stories while estimates are being made. (I am wondering if the intent of reporting the issue was to have some kind of soft save in the background, but personally I don't think this tool needs that complexity added to it either.) Changing the story has reset the votes since forever, and it isn't that hard to tell your teammates to stop doing that. We reuse sessions, and the session "owner" may not even be in the meeting. I am on a couple of teams where we can no longer use this tool at all anymore. We have loved how easily this integrates with ADO, so it sure would be nice to roll this change back. Thanks for listening.

**Comment by alizard2** [comment]
> Well, this is a breaking change. It would have been better if alizard2 had just trained the team members to not click on other user stories while estimates are being made. (I am wondering if the intent of reporting the issue was to have some kind of soft save in the background, but personally I don't think this tool needs that complexity added to it either.) Changing the story has reset the votes since forever, and it isn't that hard to tell your teammates to stop doing that. We reuse sessions, and the session "owner" may not even be in the meeting. I am on a couple of teams where we can no longer use this tool at all anymore. We have loved how easily this integrates with ADO, so it sure would be nice to roll this change back. Thanks for listening.

We did train them but it just happens sometimes.  Not sure what the issue is with fixing it if votes have already been made to prevent navigating to another story in the session or just save the votes that have been made if not revealed (soft save) or even a warning that votes will be lost when another user story is attempting to be clicked.  All good development practices IMO.

**Comment by Blackbaud-MaryRudisill** [comment]
A warning or almost anything else would have been fine. We got the update and can no longer Reveal votes because the session owner is not present and Reveal is now disabled.

**Comment by Blackbaud-MaryRudisill** [comment]
> A warning or almost anything else would have been fine. We got the update and can no longer Reveal votes because the session owner is not present and Reveal is now disabled.

And I commented too soon. I've been looking around for options and I just found that there is an option to turn this off. At least I am assuming that is what "Only creator can change work items and commit" will do. Apologies. I will see if we can find the session owner, and if not we should be able to create new sessions and move forward. Sorry! Ignore me on this. 🤦
